Storage/SQL/SQLiteHistory.swift

```diff
@@ -82,18 +82,30 @@ public class SQLiteHistory {
 
     private var ignoredSchemes = ["about"]
 
-    required public init(db: BrowserDB) {
+    required public init?(db: BrowserDB, version: Int? = nil) {
```

Comment:
I wanted a way to detect failure in tests. I wouldn't mind detecting failure in the app (and not crashing, but just not storing history), but I think we voted for this type of behavior instead (i.e. crash). Right now, we crash and you'll restart with a (hopefully working) DB? We should probably design a small UI for that sort of situation.